frontend/apps/core-app/src/components/NavHeader/LargeNavHeader.component.tsx

```diff
@@ -29,7 +32,7 @@ export const LargeNavHeaderComponent: React.FC<
         <HStack>
           <Link
             to={{
-              screen: routes.recipientsRoot.name,
+              screen: routes.recipientsList.name,
               params: { screen: routes.recipientsList.name },
```

Comment:
```suggestion
              screen: routes.recipientsList.name,
```
why is the params necessary here?